vsintegration/src/FSharp.Editor/Navigation/FindUsagesService.fs

```diff
@@ -133,7 +133,7 @@ type internal FSharpFindUsagesService
                         | _ ->
                             for referenceDocSpan in referenceDocSpans do
                                 for definitionItem in definitionItems do
-                                    let referenceItem = SourceReferenceItem(definitionItem, referenceDocSpan)
+                                    let referenceItem = SourceReferenceItem(definitionItem, referenceDocSpan, true)
```

Comment:
Note: you may want to comment that you're just passing 'true' for now until you get info on if this is actually a write or not from your compiler.